test/test_loggers.py

```diff
@@ -36,6 +36,18 @@ def tb_logger(tmp_path_factory):
     del logger
 
 
+def get_example_config():
```

Comment:
```suggestion
@fixture
def config():
```
oc we can change the name